### PR TITLE
Working add members page

### PR DESCRIPTION
**Pull request by j3ffml**
Add members page which requires user logged in and has a house.
Sends an email to each address provided with a link to join the house. that link requires authentication, so if they have an account, they will be redirected to join page, otherwise they have to register, log in and click the join link again... that can be streamlined in the future, but this works for now.

### CODE REVIEW

File: fullhouse/dashboard/views.py [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
 from django.contrib.auth.forms import AuthenticationForm
+from django.forms.formsets import formset_factory
 
-from forms import CreateAnnouncementForm
-from forms import CreateHouseForm
-from models import Announcement
-from models import House
+from forms import (
+    CreateHouseForm,
+    CreateAnnouncementForm
```
Comment (natmote) [inline fullhouse/dashboard/views.py]: Syntax error. At least run your shit before submitting a pull request.

Comment (j3ffml) [inline fullhouse/dashboard/views.py]: fixed

File: fullhouse/templates/addmembers/invite_email.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+{{ from_username }}{% trans "has invited you to join a house on" %} {{ site.name }}:
```
Comment (natmote) [inline fullhouse/templates/addmembers/invite_email.txt]: there is no space between the username and "has"

**Comment by natmote** [comment]
So, this looks pretty good, but the links are pretty lacking. For example, there's no way to get to the dashboard. from the add members page, and there's no way to get to the add members page from the dashboard.

The functionality is there, though, which is great. I'll merge it once you fix the spacing error in the email.

**Comment by natmote** [comment]
actually, I just tried to accept an invitation, and it said "invitation accept failed." Did you test this functionality?

**Comment by j3ffml** [comment]
It will fail if you accept the invite as a user that is already a member. I should change the message in that case, as it's just the general one (also fails for invalid invitation keys). To correctly test it, accept the invite from an account that isn't already a house member.

**Comment by natmote** [comment]
What I did was log out, then click the invite link. It sent me to the login page, where I clicked register. I activated my new account, then logged in with my new account, and then clicked the link, and it failed.

**Comment by natmote** [comment]
with your new changes, following the same steps, I get "invalid invitation link"

edit: I said the wrong word

**Comment by j3ffml** [comment]
did you create the new account with the same email as the one you sent the invite to? it also fails if the emails are not the same. i.e. if you sent the invite to nat@natmote.com and you try to join the house while logged in as herp@herpderp.com, it will fail, even if you have the same link.

**Comment by natmote** [comment]
ah, that's what it was. I was just bullshitting the email fields. I'll merge it.

**Pull request merged**